piv/key.go

```diff
@@ -298,6 +341,11 @@ func RetiredKeyManagementSlot(key uint32) (Slot, bool) {
 	return slot, ok
 }
 
+// String returns the two-character hex representation of the slot
+func (s *Slot) String() string {
```

Comment:
nit: while testing this locally, I noticed that vet doesn't let you format Attestation.Slot with a "%s"

```
% go test -test.v -test.run=TestYubiKeyAttestation ./piv -wipe-yubikey
# github.com/go-piv/piv-go/piv
piv/key_test.go:437:3: Errorf format %s has arg a.Slot of wrong type github.com/go-piv/piv-go/piv.Slot
FAIL	github.com/go-piv/piv-go/piv [build failed]
FAIL
```

Maybe consider having this be a method not on the pointer?

```
func (s Slot) String() string
```